exporter/observiqexporter/config.go

```diff
@@ -37,12 +38,15 @@ type Config struct {
 	APIKey string `mapstructure:"api_key"`
 	// Secret key for authenticating with the ingestion endpoint (required if no APIKey)
 	SecretKey string `mapstructure:"secret_key"`
-	// Endpoint URL; Defines the ingestion endpoint (optional)
+	// Endpoint URL; Defines the ingestion endpoint
```

Comment:
Why the redaction of optional, it looks like all these parameters are still optional no?

Reply:
My concern here is more that the component would need these values filled in if you were to construct this component programmatically. Which, is done by CreateDefaultConfig, but there's nothing forcing someone to do that. So to me, I see it as not really optional, but filled in by the defaults if you choose to use the CreateDefaultConfig function.

Is there any convention for what information should be in these config comments? I wanted to go back and clean up the comments in a future PR, anyways, because most of them aren't idiomatic Go comments.

Reply:
> Is there any convention for what information should be in these config comments?

It's a good question, there should be... From what I can tell looking at other code in this repo, it looks like the docstring in some cases includes the default value, but not always. Maybe this should be the convention, what do you think?

Reply:
I think that sounds good. I changed the docstring to include defaults (where applicable), which I think is more clear than just "optional".